.ci/go-test.sh

```diff
@@ -10,6 +10,9 @@ set -e
 script_name=${0##*/}
 typeset -A long_options
 
+# directory to use for temporary files
+TMPDIR=${TMPDIR:-/tmp}
```

Comment:
is `mktemp -d`  overkill :) ?

Reply:
you mean specifically make a new tmp directory for the tests to use @marcov ? I could go for that.